utils/terrascript_client.py

```diff
@@ -456,7 +456,17 @@ def populate_tf_resource_rds(self, resource, namespace_info,
             values['parameter_group_name'] = pg_name
 
         enhanced_monitoring = values.pop('enhanced_monitoring')
+
+        # monitoring interval should only be set if enhanced monitoring
+        # is true
+        if not enhanced_monitoring and values.get('monitoring_interval'):
```

Comment:
This will except without a default value if monitoring_interval isn't defined.

Reply:
Defaulting to `None`

Reply:
`get` does not throw an exception when the key doesn't exist, that's `pop`:
```
>>> d = {}
>>> value = d.get('key')
>>> value
>>> d.pop('key')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'key'
>>> value = d.pop('key', None)
>>> value
>>>
```